src/renderer/base/renderer.cpp

```diff
@@ -1231,6 +1239,11 @@ void Renderer::ResetErrorStateAndResume()
     EnablePainting();
 }
 
+void Microsoft::Console::Render::Renderer::UpdateLastHoveredInterval(const ThisTree::interval newInterval)

```

Comment:
nit
```suggestion
void Renderer::UpdateLastHoveredInterval(const ThisTree::interval newInterval)
```

man whatever editor you're using loves full namespaces